tests/test_init.py

```diff
@@ -44,6 +44,13 @@ def test_systemd_no_udev_present(self, auto_container):
     def test_systemd_boottime(self, auto_container):
         """Ensure the container startup time is below 5 seconds"""
 
+        # FIXME: Temporary workaround: While using emulated workers, the startup time test doesn't make sense.'
+        if auto_container.connection.system_info.arch == "ppc64le":
+            pytest.skip(
+                "boottime test temporarily disabled on emulated ppc64le workers."
+            )
+            return
```

Comment:
Bad wording on my end, you can then omit the `return`, everything after `pytest.skip()` is ignored.
```suggestion
```

Reply:
done